tardis/default_settings.py

```diff
@@ -377,14 +378,26 @@ def get_admin_media_path():
 
 # Settings for the single search box
 # Set HAYSTACK_SOLR_URL to the location of the SOLR server instance
-SINGLE_SEARCH_ENABLED = False
-HAYSTACK_SITECONF = 'tardis.search_sites'
-HAYSTACK_SEARCH_ENGINE = 'solr'
-HAYSTACK_SOLR_URL = 'http://127.0.0.1:8080/solr'
+SINGLE_SEARCH_ENABLED = True
```

Comment:
I just noticed that search is enabled by default. Probably not a good idea, @grischa.